src/main/java/core/basesyntax/servise/impl/CsvFileReaderServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.servise.impl;
+
+import core.basesyntax.servise.ReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class CsvFileReaderServiceImpl implements ReaderService {
+    public static final String FILE_PATH = "src/main/resources/file.txt";
+
+    @Override
+    public List<String> readFromFile() {
+        try (Stream<String> streamFromFile = Files.lines(Paths.get(FILE_PATH))) {
```

Comment:
let's make this solution more flexible and testable
accept filepath as an argument, move constant to Main, make it private